clientSecurity/scripts/start.gfsh

```diff
@@ -20,7 +20,7 @@
 # This requires that the example_security.json be on the classpath.
 # Recall that the --classpath option is specified relative to the locator's working directory.
 
-start locator --name=locator --bind-address=127.0.0.1\
+start locator --name=locator --bind-address=127.0.0.1 --connect=false  --enable-cluster-configuration=false\
```

Comment:
Isn't this `--enable-cluster-configuration=false` overwritten by the use of a security manager?

Reply:
I don't think we want to show examples of *not* using cluster configuration...do we?

Reply:
same comment here, we do not want to show an example with cluster configuration disabled.

Reply:
Fixed. I agree we shouldnt be showing a wrong example. I was trying to make the example work with relative paths which is the easiest way for developers to run the examples. GEODE-4332 fixes not to retrieve cluster configuration service status when auto-connect is false, with this fix we are good providing keystore relative to locator`s JVM and then a second step gfsh can connect to the locator with paths relative to gfsh jvm.